### DISCUSSION TITLE
City Center command from anywhere.

### DISCUSSION THREAD

**Discussion by Brosephine** [Arx-Game/arxcode#387] [Ideas]
Home is a great tool to return easily to your home. 

How about a CC or an ARX command that will take us straight to the City Center?

**Comment by TehomCD** [comment]
I'll talk it over with Apostate. He's a little reluctant to add too many instant-travel abilities because it reduces the possibility of people bumping into one another organically on the grid, but there's definitely arguments to be made for it.
